main.js

```diff
@@ -18,7 +18,7 @@ function promisify(func) {
       args.push(function (err) {
         if (err !== null) { return reject(err); }
         var rest = Array.prototype.slice.call(arguments, 1);
-        return resolve(rest.length === 1 ? rest[0] : rest);
+        return resolve(...rest);
```

Comment:
(pre-existing, but it seems `// Note: "this" is extracted from this scope.` is outdated and can be removed)